### PR TITLE
Add @celo/dev-utils

### PR DESCRIPTION
**Pull request by nambrot**
### Description

This PR adds `@celo/dev-utils`, a `utils` package that is meant to be used as a `devDependency`. It's primary use case is to reuse the ganache setup currently present in `cli` and `contractkit` (and soon `attestation-service`). Due to the way `jest` uses `globalSetup`, depending packages will still need to define their own setup/teardown files.

### Tested

- CI

### CODE REVIEW

**Comment by jmrossy** [comment]
Unassigning because CI tests failing

File: packages/contractkit/package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "web3-utils": "1.0.0-beta.37"
   },
   "devDependencies": {
+    "@celo/dev-utils": "0.0.1",
```
Comment (jmrossy) [inline packages/contractkit/package.json]: Can you make this version `-dev` to keep with the new scheme @mcortesi just introduced?

File: packages/contractkit/package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
     "clean:all": "yarn clean && rm -rf src/generated",
     "build:gen": "yarn --cwd ../protocol build",
     "prepublishOnly": "yarn build:gen && yarn build",
-    "test:prepare": "yarn --cwd ../protocol devchain generate .devchain --migration_override src/test-utils/migration-override.json",
+    "test:prepare": "yarn --cwd ../protocol devchain generate .devchain --migration_override ../dev-utils/src/migration-override.json",
```
Comment (jmrossy) [inline packages/contractkit/package.json:23]: @mcortesi Should this be `pretest` instead of `test:prepare`?

**Review by jmrossy — APPROVED** [review]
Looks good to me, just wondering about the version number

File: packages/contractkit/package.json (hunk 1 above)
Comment (nambrot) [reply packages/contractkit/package.json]: Will do, though I imagine this is not a package we ever have to publish to npm i think?

File: packages/contractkit/package.json (hunk 2 above)
Comment (nambrot) [reply packages/contractkit/package.json:23]: I believe this command is expected to run before every test which is why it is not named `pretest`

**Pull request merged**